packages/vibrant-components/src/lib/ModalBottomSheet/ModalBottomSheetMenu.tsx

```diff
@@ -0,0 +1,19 @@
+import { cloneElement } from 'react';
+import { Box, PressableBox } from '@vibrant-ui/core';
+import { Body } from '../Body';
+import { HStack } from '../HStack';
+import { withModalBottomSheetMenuVariation } from './ModalBottomSheetMenuProps';
+
+export const ModalBottomSheetMenu = withModalBottomSheetMenuVariation(
+  ({ rightContents, icon, onClick, title, color }) => (
```

Comment:
버튼같은 컴포넌트 보면 보통은 ReactElement를 안 받고, `IconComponent` props로 받은 다음 cloneElement 안하게 했던 것 같아요. 

```suggestion
  ({ rightContents, IconComponent, onClick, title, color }) => (
```

Reply:
반영했습니다 !